k8s/agones/Makefile

```diff
@@ -0,0 +1,112 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+TAG ?= 0.8.1
+DEMO_RELEASE_TAG ?= release-0.8.0
+NAME ?= agones
+
+
+$(info ---- TAG = $(TAG))
+$(info ---- DEMO_RELEASE_TAG = $(DEMO_RELEASE_TAG))
+
+
+# source repo and destination tags for images
+SOURCE_REGISTRY ?= gcr.io/agones-images
+APP_DEPLOYER_IMAGE ?= $(REGISTRY)/agones/deployer:$(TAG)
+AGONES_CONTROLLER_IMAGE ?= $(REGISTRY)/agones:$(TAG)
+AGONES_SDK_IMAGE ?= $(REGISTRY)/agones/agones-sdk:$(TAG)
+AGONES_PING_IMAGE ?= $(REGISTRY)/agones/agones-ping:$(TAG)
+
+
+XONOTIC_DEMO_BASE=https://github.com/GoogleCloudPlatform/agones/raw/$(DEMO_RELEASE_TAG)/examples/xonotic
+
+
+APP_PARAMETERS ?= { \
+    "APP_INSTANCE_NAME": "$(NAME)", \
+    "NAMESPACE": "$(NAMESPACE)" \
+}
+
+
+APP_TEST_PARAMETERS ?= { \
+}
+
+
+app/build:: .build/agones/helmdeps \
+            .build/agones/xonotic.yaml \
+            .build/agones/deployer \
+            .build/agones/agones-controller \
+            .build/agones/agones-sdk \
+            .build/agones/agones-ping
+
+
+.build/agones: | .build
+	mkdir -p "$@"
+
+
+.build/agones/helmdeps: chart/agones-mp/requirements.* \
+                        | .build/agones
+	-helm init --client-only
```

Comment:
The dash before `helm` command looks like a typo. Is it?

Reply:
No, it's to ensure that if the command fails, it will still continue execution. I wasn't sure in what state Cloud Builder would be when I run that command, so I wanted to ensure that if it was already initialized, it wouldn't abort the rest of the steps.